Formula/circleci.rb

```diff
@@ -3,8 +3,8 @@ class Circleci < Formula
   homepage "https://circleci.com/docs/2.0/local-cli/"
   # Updates should be pushed no more frequently than once per week.
```

Comment:
We should probably hold off. The last update was 3 days ago (#61942)

Reply:
It actually populates another two PRs today, maybe we should consider remove that line? (there was a time that circleci has less release every day)
#62129 #62132

Reply:
I am going to pull the usage report.